plugins/meta/portmap/main.go

```diff
@@ -89,12 +90,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 		if err := forwardPorts(netConf, netConf.ContIPv4); err != nil {
 			return err
 		}
+		// Delete conntrack entries for UDP to avoid conntrack blackholing traffic
+		// due to stale connections. We do that after the iptables rules are set, so
+		// the new traffic picks them
+		if err := deletePortmapStaleConnections(netConf.RuntimeConfig.PortMaps, unix.AF_INET); err != nil {
+			return err
```

Comment:
I know its unlikely, but we should probably just output the error to stderr and continue; if we fail to delete conntrack, it shouldn't block container startup.